### PR TITLE
Bump spring-boot from 2.6.7 to 2.7.1

### PR DESCRIPTION
**Pull request by tokuhirom**

### CODE REVIEW

File: config/findbugs/excludeFilter.xml [hunk 1]
```diff
@@ -14,6 +14,12 @@
     <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" />
   </Match>
 
+  <!-- Ignore false positive -->
```
Comment (tokuhirom) [inline config/findbugs/excludeFilter.xml:17]: maybe it's false positive.
https://github.com/spotbugs/spotbugs/issues/782

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ public CompletableFuture<MessageContentResponse> getRichMenuImage(final String r
     @Override
     public CompletableFuture<BotApiResponse> setRichMenuImage(
             final String richMenuId, final String contentType, final byte[] content) {
-        final RequestBody requestBody = RequestBody.create(MediaType.parse(contentType), content);
+        final RequestBody requestBody = RequestBody.create(content, MediaType.parse(contentType));
```
Comment (tokuhirom) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java:49]: The old style was deprecated. We should follow new argument order.

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java [hunk 3]
```diff
@@ -85,12 +85,20 @@ public void onFailure(final Call<ResponseBody> call, final Throwable t) {
         }
 
         private MessageContentResponse convert(final Response<ResponseBody> response) {
+            ResponseBody body = response.body();
```
Comment (tokuhirom) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java:88]: Add some null value detection to suppress spotbugs.

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java [hunk 4]
```diff
@@ -85,12 +85,20 @@ public void onFailure(final Call<ResponseBody> call, final Throwable t) {
...
+            if (body == null) {
```
Comment (dayflower) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java:89]: I guess you can use [Objects.requireNonNull()](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-java.lang.String-)
(but not sure spotbugs can detect that)

Comment (tokuhirom) [reply line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java:89]: good catch. i'll try it.

**Review by dayflower — APPROVED** [review]
looks great

**Pull request merged**